checkov/sca_image/runner.py

```diff
@@ -246,6 +248,31 @@ def get_image_report(self, dockerfile_path: str, image: Image, runner_filter: Ru
 
         return report
 
+    def get_license_statuses(self, packages: list[dict[str, Any]]) -> List[_LicenseStatus]:
+        requests_input = [
+            {"name": package.get("name", ""), "version": package.get("version", ""), "lang": package.get("type", "")}
+            for package in packages
+        ]
+        get_license_violation_url = f"{bc_integration.api_url}/api/v1/vulnerabilities/packages/get-licenses-violations"
```

Comment:
Can we put it as a class parameter instead?

Reply:
sure! I will fix